sqlobject/src/main/java/org/jdbi/v3/sqlobject/statement/internal/ResultReturner.java

```diff
@@ -106,15 +106,34 @@ private static Object checkResult(Object result, Type type) {
         return result;
     }
 
+    static class VoidReturner extends ResultReturner {
```

Comment:
why did this move up suddenly?

Reply:
I re-ordered them to be in the same order as their usage in the methods higher up. I think that was a thing last time, some classes were in order of usage and got reordered to be out-of-order, which wasn't appreciated...